pom.xml

```diff
@@ -2136,6 +2136,41 @@
             </properties>
         </profile>
 
+        <profile>
+            <id>java-21</id>
+            <activation>
+                <jdk>21</jdk>
+            </activation>
+            <properties>
+                <java.version>21</java.version>
+                <maven.compiler.source></maven.compiler.source>
+                <maven.compiler.target></maven.compiler.target>
+                <maven.compiler.release>${java.version}</maven.compiler.release>
+                <maven.plugin.surefire.argLine>-XX:+IgnoreUnrecognizedVMOptions
```

Comment:
actually, we can set this property unconditionally, instead of duplicating it in different profiles

Reply:
this is done in 5623ead6bdea8a9b5948c07ca48c857c2e518940